main.cpp

```diff
@@ -369,6 +376,15 @@ static int parse_conf(const char *conf_file_name)
         opt.logs_dir = value;
     }
 
+    value = conf.next_from_section("General", "verbose");
```

Comment:
I think it would be better to allow setting the log level on configuration and parse the argument as a string.

LogLevel = error | warning | info | debug